eventual-cassandra/src/test/scala/com/evolutiongaming/kafka/journal/eventual/cassandra/EventualCassandraSpec.scala

```diff
@@ -34,7 +34,7 @@ class EventualCassandraSpec extends EventualJournalSpec {
           val journals = journalsOf(segmentSize, delete, segmentsFirst = segmentsFirst, segmentsSecond = segmentsSecond)
           val (_, result) = test(journals)
             .run(State.empty)
-            .get
+            .unsafeRunSync()
```

Comment:
overall I'd prefer to not use `unsafeRunSync` without timeouts or even would better go completely `async`

Reply:
Noted, I will try to move them to async when the initial work is done and tests pass at least